chemprop/cli/utils/MAB_parsing.py

```diff
@@ -212,7 +212,15 @@ def build_MAB_data_from_files(
 
     datapoints = [
         MolAtomBondDatapoint(
-            mol=mols[i],
+            smiles=smis[i],
+            _keep_h=make_mol_kwargs["keep_h"] if "keep_h" in make_mol_kwargs else False,
+            _add_h=make_mol_kwargs["add_h"] if "add_h" in make_mol_kwargs else False,
+            _ignore_stereo=make_mol_kwargs["ignore_stereo"]
+            if "ignore_stereo" in make_mol_kwargs
+            else False,
+            _reorder_atoms=make_mol_kwargs["reorder_atoms"]
+            if "reorder_atoms" in make_mol_kwargs
+            else False,
```

Comment:
All of these attempted key accesses will result in a KeyError if they key is not present because the if else is executed greedily (I think) so we need different approach (which is also conveniently a bit more concise):

```suggestion
            _keep_h=make_mol_kwargs.get("keep_h", False),
            _add_h=make_mol_kwargs.get("add_h", False),
            _ignore_stereo=make_mol_kwargs.get("ignore_stereo", False),
            _reorder_atoms=make_mol_kwargs.get("reorder_atoms", False),
```